fabfile.py

```diff
@@ -762,6 +772,13 @@ def reset_security_tokens():
     reset_controller_token()
     config_couchdb()
     print(green('All the tokens have been reset.'))
+    
+    if is_pi():
+        reset_ssh_keys()
```

Comment:
There is already a function for that: https://github.com/cozy/cozy-setup/blob/master/fabfile.py#L755

I think you should put your reset here.